specs/phase0/fork-choice.md

```diff
@@ -362,6 +391,14 @@ def on_block(store: Store, signed_block: SignedBeaconBlock) -> None:
     # Add new state for this block to the store
     store.block_states[hash_tree_root(block)] = state
 
+    # Add proposer score boost if the block is timely
+    if (get_current_slot(store) == block.slot and
```

Comment:
Generally prefer single line `if`s when possible. Easy way to achieve it is to set an intermediate bool var

```suggestion
    # Add proposer score boost if the block is timely
    is_before_attestation_broadcast = store.time % SECONDS_PER_SLOT < SECONDS_PER_SLOT // ATTESTATION_OFFSET_QUOTIENT
    if get_current_slot(store) == block.slot and is_before_attestation_broadcast:
```